index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="head">
+      <h2>Welcome</h2>
+      <img src="./Icons/Unionburger_icon.png" alt="" />
```

Comment:
- [ ] Don't you think the menu button is responsible for the navigation manu? so let's put this in a nav tag.


- [ ] The icon image is not added to the repository, it's recommended to include it in your repository so that we can understand how your design looks like.